internal/utils/errors.go

```diff
@@ -5,6 +5,15 @@ import (
 	"strings"
 )
 
+// Common i/o related errors
```

Comment:
The reason I put them there is because they are not our own errors. If you search for them you won't find a place in our code where they are returned. So since they are just declared here out of utility I thought a dedicated spot makes sense.

Reply:
Hm ok. It's not a `util` though but I'll get over it.